fixture-monkey/src/main/java/com/navercorp/fixturemonkey/FixtureMonkeyBuilder.java

```diff
@@ -498,21 +495,10 @@ public FixtureMonkeyBuilder enableLoggingFail(boolean enableLoggingFail) {
 	}
 
 	public FixtureMonkeyBuilder useExpressionStrictMode() {
-		this.monkeyExpressionFactory = expression -> {
-			MonkeyExpression monkeyExpression = new ArbitraryExpressionFactory().from(expression);
-
-			return new MonkeyExpression() {
-				@Override
-				public NodeResolver toNodeResolver() {
-					return new ApplyStrictModeResolver(monkeyExpression.toNodeResolver());
-				}
-
-				@Override
-				public List<NextNodePredicate> toNextNodePredicate() { // TODO:
-					return monkeyExpression.toNextNodePredicate();
-				}
-			};
-		};
+		this.monkeyExpressionFactory = new StrictModeMonkeyExpressionFactory(
+			new ArbitraryExpressionFactory(),
+			fixtureMonkeyOptionsBuilder.getDefaultPropertyNameResolver()
```

Comment:
I think it is better to add `propertyNameResolvers` in FixtureMonkeyBuilder as well. 

```
private List<MatcherOperator<PropertyNameResolver>> propertyNameResolvers = new ArrayList<>();
```

All options, such as `pushAssignableTypePropertyNameResolver`, use propertyNameResolvers from FixtureMonkeyBuilder and pass them to FixtureMonkeyOptionsBuilder within the build.